pkg/manifests/manifests_test.go

```diff
@@ -1194,6 +1194,74 @@ ingress:
 	}
 }
 
+func TestPrometheusQueryLogFileConfig(t *testing.T) {
```

Comment:
the test can also check the presence/absence of the volume + volume mount.

Reply:
Added tests to volumes